runtime/test-fw/src/mbox_responder.rs

```diff
@@ -1,26 +1,91 @@
 // Licensed under the Apache-2.0 license
 
-#![no_main]
-#![no_std]
+#![cfg_attr(not(feature = "std"), no_std)]
+#![cfg_attr(not(feature = "std"), no_main)]
 
 use core::mem::size_of;
 
-use caliptra_common::mailbox_api::CommandId;
+use caliptra_common::{handle_fatal_error, mailbox_api::CommandId};
 use caliptra_drivers::{
+    cprintln,
     pcr_log::{PCR_ID_STASH_MEASUREMENT, RT_FW_JOURNEY_PCR},
-    Array4x12,
+    Array4x12, CaliptraError, CaliptraResult,
 };
 use caliptra_registers::{mbox::enums::MboxStatusE, soc_ifc::SocIfcReg};
-use caliptra_runtime::{ContextState, DpeInstance, Drivers, U8Bool, MAX_HANDLES};
+use caliptra_runtime::{ContextState, DpeInstance, Drivers, RtBootStatus, U8Bool, MAX_HANDLES};
 use caliptra_test_harness::{runtime_handlers, test_suite};
 use zerocopy::{AsBytes, FromBytes};
 
 const FW_LOAD_CMD_OPCODE: u32 = CommandId::FIRMWARE_LOAD.0;
 
-fn mbox_responder() {
-    let mut drivers = unsafe { Drivers::new_from_registers().unwrap() };
-    assert!(drivers.persistent_data.get().fht.is_valid());
-    let mut mbox = drivers.mbox;
+#[cfg(feature = "std")]
+pub fn main() {}
```

Comment:
In what circumstances would this be build with `std` enabled? Especially since if it is, I think it wouldn't do anything since the main implementation is empty.

Reply:
Removed.